proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for correct receiving of system time.
+
+
+## Motivation
+
+Allow SDL Core obtain time from the system in UTC format. Exclude possibility of UTC time distortion in case using system time from other components of the system.
+
+
+## Proposed solution
+
+
+The solution proposes adding new RPC between SDL and HMI.  GetSystemTime gives possibility to get the System Time status by making the following additions to the HMI API.
+
+SDL Core must send this GetSystemTime RPC to HMI in case of Policy Table Update procedure and before Security TLS Handshake.
+
+Expected that HMI will request system time from the system and provide to SDL Core via GetSystemTime_response

```

Comment:
@AStasiuk please remove underscope in `GetSystemTime_response`

Reply:
@AKalinich-Luxoft Fixed in 475a8bf